runtime panic issue with 0.3.0 release

Still have some issue with running rover in my local

### How to reproduce

- grab the artifact from [releases page](https://github.com/im2nguyen/rover/releases/tag/v0.3.0)
- create `main.tf`
- run `rover_0.3.0_darwin_arm64/rover_v0.3.0 -standalone -tfPath <path-to-terraform-installation>`

#### main.tf ref

```tf
output "hello_world" {
  value = "Hello, World!"
}
```

### error log
```sh
2022/01/24 14:25:15 Starting Rover...
2022/01/24 14:25:15 Initializing Terraform...
2022/01/24 14:25:15 Generating plan...
2022/01/24 14:25:15 Generating resource overview...
open .terraform/modules/modules.json: no such file or directory
2022/01/24 14:25:15 Generating resource map...
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0xb0 pc=0x10302d520]

goroutine 1 [running]:
main.(*rover).GenerateModuleMap(0x1400000b320, 0x140003c9b68, {0x0, 0x0})
	rover/map.go:103 +0x1e0
main.(*rover).GenerateMap(0x1400000b320)
	rover/map.go:322 +0x1c8
main.(*rover).generateAssets(0x1400000b320)
	rover/main.go:193 +0x10c
main.main()
	rover/main.go:138 +0xeb8
```

--- a/example/simple-test/main.tf
+++ b/example/simple-test/main.tf
@@ -1,0 +1,3 @@
+output "hello_world" {
+  value = "Hello, World!"
+}

--- a/graph.go
+++ b/graph.go
@@ -244,10 +244,6 @@ func (r *rover) addEdges(base string, parent string, edgeMap map[string]Edge, re
 
 		configId := matchBrackets.ReplaceAllString(id, "")
 
-		if _, ok := r.RSO.States[id]; ok {
-			configId = r.RSO.States[id].ConfigId
-		}
-
 		var expressions map[string]*tfjson.Expression
 
 		if r.RSO.Configs[configId] != nil {

--- a/rso.go
+++ b/rso.go
@@ -23,7 +23,6 @@ type ResourcesOverview struct {
 // ResourceOverview is a modified tfjson.Plan
 type StateOverview struct {
 	// ChangeAction tfjson.Actions        `json:change_action`
-	ConfigId  string                    `json:"config_id,omitempty"`
 	Change    tfjson.Change             `json:"change,omitempty"`
 	Module    *tfjson.StateModule       `json:"module,omitempty"`
 	DependsOn []string                  `json:"depends_on,omitempty"`
@@ -60,7 +59,7 @@ func (r *rover) PopulateModuleLocations(moduleJSONFile string, locations map[str
 
 	jsonFile, err := os.Open(moduleJSONFile)
 	if err != nil {
-		fmt.Println(err)
+		log.Println("No submodule configurations found...")
 	}
 	defer jsonFile.Close()
 
@@ -145,7 +144,7 @@ func (r *rover) PopulateConfigs(parent string, parentKey string, rso *ResourcesO
 		if !child.Diagnostics.HasErrors() {
 			rc[mn].Module = child
 		} else {
-			fmt.Printf("Continuing without loading module from filesystem: %s\n", childKey)
+			log.Printf("Continuing without loading module from filesystem: %s\n", childKey)
 		}
 
 		rc[mn].ModuleConfig = m
@@ -155,7 +154,6 @@ func (r *rover) PopulateConfigs(parent string, parentKey string, rso *ResourcesO
 }
 
 func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.StateModule, prior bool) {
-	matchBrackets := regexp.MustCompile(`\[[^\[\]]*\]`)
 	childIndex := regexp.MustCompile(`\[[^[\]]*\]$`)
 
 	rs := rso.States
@@ -166,13 +164,11 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 		parent := module.Address
 		//fmt.Printf("ID: %v\n", id)
 		if rst.AttributeValues != nil {
-			configId := matchBrackets.ReplaceAllString(id, "")
 
 			// Add resource to parent
 			// Create resource if doesn't exist
 			if _, ok := rs[id]; !ok {
 				rs[id] = &StateOverview{}
-				rs[id].ConfigId = configId
 				if rst.Mode == "data" {
 					rs[id].Type = ResourceTypeData
 				} else {
@@ -182,6 +178,8 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 
 			if _, ok := rs[parent]; !ok {
 				rs[parent] = &StateOverview{}
+				rs[parent].Type = ResourceTypeModule
+				rs[parent].IsParent = false
 				rs[parent].Children = make(map[string]*StateOverview)
 			}
 
@@ -199,7 +197,6 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 
 				rs[module.Address].Children[parent] = rs[parent]
 
-				rs[parent].ConfigId = matchBrackets.ReplaceAllString(parent, "")
 			}
 
 			//fmt.Printf("%v - %v\n", id, parent)
@@ -224,19 +221,15 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 
 	for _, childModule := range module.ChildModules {
 
-		matchBrackets := regexp.MustCompile(`\[[^\[\]]*\]`)
-
 		parent := module.Address
-		parentConfig := matchBrackets.ReplaceAllString(parent, "")
 
 		id := childModule.Address
-		configId := matchBrackets.ReplaceAllString(id, "")
 
 		if _, ok := rs[parent]; !ok {
 			rs[parent] = &StateOverview{}
 			rs[parent].Children = make(map[string]*StateOverview)
-			rs[parent].ConfigId = parentConfig
 			rs[parent].Type = ResourceTypeModule
+			rs[parent].IsParent = false
 		}
 
 		if childIndex.MatchString(id) {
@@ -246,7 +239,6 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 			if _, ok := rs[parent]; !ok {
 				rs[parent] = &StateOverview{}
 				rs[parent].Children = make(map[string]*StateOverview)
-				rs[parent].ConfigId = configId
 				rs[parent].Type = ResourceTypeModule
 				rs[parent].IsParent = true
 			}
@@ -261,7 +253,6 @@ func (r *rover) PopulateModuleState(rso *ResourcesOverview, module *tfjson.State
 		if _, ok := rs[id]; !ok {
 			rs[id] = &StateOverview{}
 			rs[id].Children = make(map[string]*StateOverview)
-			rs[id].ConfigId = configId
 			rs[id].Type = ResourceTypeModule
 		}
 
@@ -300,8 +291,8 @@ func (r *rover) GenerateResourceOverview() error {
 	if !rootModule.Diagnostics.HasErrors() {
 		rc[""].Module = rootModule
 	} else {
-		fmt.Printf("Could not load configuration from: %v\n", r.WorkingDir)
-		fmt.Printf("Continuing without configuration file data...")
+		log.Printf("Could not load configuration from: %v\n", r.WorkingDir)
+		log.Printf("Continuing without configuration file data...")
 	}
 
 	rc[""].ModuleConfig = &tfjson.ModuleCall{}
@@ -323,6 +314,14 @@ func (r *rover) GenerateResourceOverview() error {
 		if r.Plan.PlannedValues.RootModule != nil {
 			r.PopulateModuleState(rso, r.Plan.PlannedValues.RootModule, false)
 		}
+	}
+
+	// Create root module in state if doesn't exist
+	if _, ok := rs[""]; !ok {
+		rs[""] = &StateOverview{}
+		rs[""].Children = make(map[string]*StateOverview)
+		rs[""].IsParent = false
+		rs[""].Type = ResourceTypeModule
 	}
 
 	// reIsChild := regexp.MustCompile(`^\w+\.\w+[\.\[]`)

